phoenix-core/src/main/java/org/apache/phoenix/coprocessor/HashJoinRegionScanner.java

```diff
@@ -74,18 +75,18 @@
     private final boolean addArrayCell;
 
     @SuppressWarnings("unchecked")
-    public HashJoinRegionScanner(RegionScanner scanner, TupleProjector projector,
+    public HashJoinRegionScanner(RegionScanner scanner, Scan scan, TupleProjector projector,
                                  HashJoinInfo joinInfo, ImmutableBytesPtr tenantId,
                                  RegionCoprocessorEnvironment env, boolean useQualifierAsIndex,
                                  boolean useNewValueColumnQualifier)
         throws IOException {
 
-        this(env, scanner, null, null, projector, joinInfo,
+        this(env, scanner, scan,null, null, projector, joinInfo,
              tenantId, useQualifierAsIndex, useNewValueColumnQualifier);
     }
 
     @SuppressWarnings("unchecked")
-    public HashJoinRegionScanner(RegionCoprocessorEnvironment env, RegionScanner scanner,
+    public HashJoinRegionScanner(RegionCoprocessorEnvironment env, RegionScanner scanner, Scan scan,
                                  final Set<KeyValueColumnExpression> arrayKVRefs,
```

Comment:
Is Scan parameter used anywhere? I could not find where you are using this one?

Reply:
I also do not see where HashJoinRegionScanner uses the new scan parameter.

Reply:
Good catch. I reverted the code.